girder_style/templates/account/base.html

```diff
@@ -5,17 +5,18 @@
 {% block body %}
 <body class="bg-gray-100 flex flex-col h-full relative">
 
-<div class="absolute bg-black bottom-0 left-0 overflow-hidden right-0 top-0 z-0">
+<div class="absolute bg-black bottom-0 left-1/4 overflow-hidden right-0 bottom-0 top-0 z-0">
   <img
-      src="https://images.pexels.com/photos/53176/architecture-iron-steel-building-53176.jpeg?cs=srgb&dl=pexels-pixabay-53176.jpg&fm=jpg"
-      class="object-cover opacity-25 w-full"
+      src="https://images.unsplash.com/photo-1543996991-8e851c2dc841?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=2251&q=80"
```

Comment:
I think we should go with an image which is a little more generic and boring. I could see lots of downstream projects' users being confused as to why there are cars on their login page. Maybe a patterned background of some sort? Ideally one which compresses well and scales to all screen sizes.